SR2021/2021-01-19-league-1-absent.md

```diff
@@ -0,0 +1,31 @@
+---
+to: Teams who didn't attend session 1
+subject: Student Robotics League 1
+---
+
+Hello there!
+
+We noticed at least one of your teams was absent from our first league session last weekend. We wanted to reach out to see if there was any more support we could provide you, or questions we could answer?
+
+## Assistance
+
+If you're struggling with your robot, here's a few resources which might help you out:
+
+- [The Docs](https://studentrobotics.org/docs/)
+- [Discord](https://studentrobotics.org/docs/team_admin/discord), to get help from other teams / blue shirts
+
+## Mentors
+
+Alternatively, if you'd prefer some more hands-on support, we may be able to allocate you a mentor.
+
+Your mentor will help guide your team towards good solutions for their robot, provide assistance where they might need it as well as help them understand the kit, the rules and the competition as a whole.
+
+Mentors are volunteers who have typically competed or helped out with the competition before. They’re able to give help on all aspects of programming your robot, and if they don’t know something, they typically know someone who does.
+
+If you're interested in getting a mentor, please let us know when works best for you. For safeguarding reasons, either you or another responsible adult such as a parent / guardian will need to be around for any video calls.
```

Comment:
Should we explicitly mention how teams should do this? Having an explicit call to action (even if it's just "reply to this email") often helps clarify.